components/stripe/actions/list-invoices/list-invoices.mjs

```diff
@@ -29,15 +29,23 @@ export default {
       ],
     },
     status: {
-      propDefinition: [
-        app,
-        "invoice_status",
+      type: "string",
+      label: "Status",
+      description: "Only return invoices that are in the given status otherwise all invoices are returned.",
```

Comment:
```suggestion
      description: "If set, only returns invoices that are in the given status. Otherwise, all invoices are returned.",
```
Text suggestion